config/default.yaml

```diff
@@ -25,7 +25,8 @@ services:
   hmctsHomePage:
     url: 'https://www.gov.uk/government/organisations/hm-courts-and-tribunals-service'
   documentManagement:
-    url: 'http://ccd-case-document-am-api-aat.service.core-compute-aat.internal' 
+    url: 'http://fis-cos-api-pr-62.service.core-compute-preview.internal' 
```

Comment:
This needs to be AAT URL in place of preview, OK if you have added temporality for testing purpose